includes/functions/functions_products.php

```diff
@@ -27,6 +27,10 @@ function zen_get_product_details($product_id, $language_id = null)
             LEFT JOIN " . TABLE_PRODUCTS_DESCRIPTION . " pd ON (p.products_id = pd.products_id AND pd.language_id = " . (int)$language_id . ")
             WHERE p.products_id = " . (int)$product_id;
     return $db->Execute($sql, 1, true, 900);
```

Comment:
leaving this line (29) will ensure that your new code will never get executed.

Reply:
late night shoddy merging (again), apologies